recipes/opensearch-dsl/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "opensearch-dsl" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opensearch-dsl-{{ version }}.tar.gz
+  sha256: 745fd4b05934b764f3ebca0dd9b1e7bd9b5934fcbb6c620d7087d24bcc8241c7
+
+build:
+  number: 0
+  skip: true   # [py==30 or py==31 or py==32 or py==33]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - ipaddress  # [py<33]
+    - opensearch-py >=1.0.0
+    - python
+    - python-dateutil
+    - six
+
+test:
+  imports:
+    - opensearch_dsl
+    - opensearch_dsl.response
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/opensearch-project/opensearch-dsl-py
+  summary: Python client for OpenSearch
+  license: Apache-2.0
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - rpozzi
```

If you don't mind having me here too :smile:
```suggestion
    - chrisburr
    - rpozzi
```